format/format.c

```diff
@@ -137,15 +133,16 @@ M0_INTERNAL int m0_format_footer_verify_generic(
 	M0_PRE(footer != NULL);
 
 	if (footer->ft_magic != M0_FORMAT_FOOTER_MAGIC) {
-		format_iem_post("format footer magic mismatch,",
-				(uint64_t)M0_FORMAT_FOOTER_MAGIC,
-				footer->ft_magic, iem);
+		format_iem_post("Internal data appears to be inconsistent and "
```

Comment:
In this case, do we need to log something into m0trace log?
For example, the expected "M0_FORMAT_FOOTER_MAGIC" and the "footer->ft_magic",
and other information about the buffer.

So, when Seagate support analyzes or inspects this issue, more information can be used to locate the data.

Reply:
OK